src/main/java/org/veupathdb/service/eda/ds/plugin/AbstractPlugin.java

```diff
@@ -321,6 +321,24 @@ protected String getViewportAsRString(XAxisViewport viewport, String xVarType) {
     }
   }
 
+  // there is probably some JRI util that would make this unnecessary if i were more clever??
+  protected String listToRVector(List<String> values) {
+    boolean first = true;
+    String vector = "c(";
+
+    for (String value : values) {
+      if (first) {
+        vector = vector + singleQuote(value);
+        first = false;
+      } else {
+        vector = vector + singleQuote(value) + ",";
```

Comment:
should be `vector + "," + singleQuote(value)` ? I think right now we'd get "('a''b','c',)"